src/OutputWriters/bit_rounding.jl

```diff
@@ -0,0 +1,52 @@
+using BitInformation
+
+struct BitRounding{K}
+    keepbits :: K
+end
+
+# number of keepbits (mantissa bits) for each variable
+default_bit_rounding(::Val{name}) where name = 23   # single precision default
+default_bit_rounding(::Val{:u}) = 2
+default_bit_rounding(::Val{:v}) = 2
+default_bit_rounding(::Val{:w}) = 2
+default_bit_rounding(::Val{:T}) = 7
+default_bit_rounding(::Val{:S}) = 16                # 12 at the surface, 16 deep ocean
+default_bit_rounding(::Val{:η}) = 6
+
+function BitRounding(outputs = nothing;
+                     user_rounding...)
```

Comment:
For the purpose of figuring out good defaults perhaps we should include `model` as an input here?

Reply:
Then `default_bit_rounding` can take `model` as an argument, and dispatch on various things, for example the equation of state (which should know the units of temperature), and perhaps the biogeochemistry model, which may know the units of some important tracers